packages/features/users/components/UserTable/BulkActions/EventList.tsx

```diff
@@ -0,0 +1,193 @@
+import type { Table } from "@tanstack/react-table";
+import { Check } from "lucide-react";
+import type { Dispatch, SetStateAction } from "react";
+import { useState } from "react";
+
+import classNames from "@calcom/lib/classNames";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { SchedulingType } from "@calcom/prisma/enums";
+import { trpc } from "@calcom/trpc";
+import type { RouterOutputs } from "@calcom/trpc/react";
+import {
+  Button,
+  Command,
+  CommandEmpty,
+  CommandGroup,
+  CommandInput,
+  CommandItem,
+  CommandList,
+  Popover,
+  showToast,
+  PopoverContent,
+  PopoverTrigger,
+} from "@calcom/ui";
+
+import type { User } from "../UserListTable";
+
+interface Props {
+  table: Table<User>;
+  teams: RouterOutputs["viewer"]["organizations"]["getTeams"] | undefined;
+}
+
+export function EventListBulkAction({ table, teams }: Props) {
+  const { t } = useLocale();
+  const utils = trpc.useUtils();
+  const teamIds = teams?.map((team) => team.id);
+  const { data } = trpc.viewer.eventTypes.getByViewer.useQuery({
+    filters: { teamIds, schedulingType: SchedulingType.ROUND_ROBIN },
```

Comment:
@PeerRich you created the issue - why are we requesting ability to assign people from org members specifically to round robin? Why not also collective?

We call this component `EventList` but really it's `RoundRobinList` - to me it makes sense to also include collective event types not only round robin.